src/contexts/TransactionsFiltersContext/TransactionsFiltersProvider.tsx

```diff
@@ -0,0 +1,39 @@
+import { ReactNode, useMemo } from 'react'
+import {
+  TransactionsFiltersContext,
+  TransactionsFiltersContextType,
+} from './TransactionsFiltersContext'
+import {
+  useTransactionsFilters,
+  UseTransactionsFiltersParams,
+} from './useTransactionsFilters'
+import { AccountItem } from '../../hooks/useBankTransactions/types'
+
+type TransactionsFiltersProviderProps = {
+  children: ReactNode
+  accountsList: AccountItem[]
+} & UseTransactionsFiltersParams
+
+export const TransactionsFiltersProvider = ({
```

Comment:
Let's prefix this (and the context) everywhere with `Bank`, because every component and context related to bank transactions starts with `BankTransactions`.
```suggestion
export const BankTransactionsFiltersProvider = ({
```